content/documentation/pages/3-stream-developer-guides/5-troubleshooting/1-debugging-stream-apps.md

```diff
@@ -203,3 +203,9 @@ graph TD;
 As discussed before the application may run as expected from your local platform however, it still fails when launched from Spring Cloud Data Flow.
 This could be because of how the container is created by the tool that is being used to create the container, for example: DockerFile, Spring Boot container Plugin, Jib, etc.
 The chart above provides a guide on how to trouble shoot why apps may fail to launch because of container related issues.
+
+[[note]]
+| Data Flow establishes some metrics properties using the values in other properties. These are denoted using `${}` for example:
```

Comment:
Instead of 

> Data Flow establishes some metrics properties using the values in other properties. These are denoted using `${}` for example: 

I would suggest something more descriptive and concrete, like:

> Data Flow uses the [Common application properties](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#_common_application_properties) and the [Spring Boot Common Tags](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-metrics-common-tags) to apply common metrics tags to all deployed `Stream` applications. Those properties often use [Spring placeholders](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-placeholders-in-properties) to resolve their values, for example: